src/app/GitUI/CommandsDialogs/FormRemotes.cs

```diff
@@ -378,6 +378,10 @@ private void application_Idle(object sender, EventArgs e)
 
         private void btnRemoteColor_Click(object sender, EventArgs e)
         {
+            colorDialog.Color = btnRemoteColor.BackColor == Color.Transparent
+                ? Color.Black
```

Comment:
```suggestion
                ? AppColor.RemoteBranch.GetThemeColor()
```
would be more helpful.

Reply:
Done